packages/config/config/devices/0x027a/templates/zooz_template.json

```diff
@@ -252,6 +276,88 @@
 			}
 		]
 	},
+	"external_switch_type_0-3": {
+		"label": "External Switch Type",
+		"valueSize": 1,
+		"unsigned": true,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 2,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle switch",
+				"value": 0
+			},
+			{
+				"label": "Momentary switch",
+				"value": 1
+			},
+			{
+				"label": "Toggle switch with fixed actions",
+				"value": 2
+			},
+			{
+				"label": "3-way impulse control",
+				"value": 3
+			}
+		]
+	},
+	"external_switch_type_0-4": {
+		"$import": "#external_switch_type_0-3",
+		"maxValue": 4,
+		"options": [
+			{
+				"label": "Toggle switch",
+				"value": 0
+			},
+			{
+				"label": "Momentary switch",
+				"value": 1
+			},
+			{
+				"label": "Toggle switch with fixed actions",
+				"value": 2
+			},
+			{
+				"label": "3-way impulse control",
+				"value": 3
+			},
+			{
+				"label": "Garage door mode (simulate momentary)",
+				"value": 4
+			}
+		]
+	},
+	"impulse_duration": {
+		"label": "Impulse Duration",
+		"description": "Choose the duration of the impulse if you're using this relay with 3-way impulse control",
+		"valueSize": 1,
+		"minValue": 2,
+		"maxValue": 200,
+		"defaultValue": 10,
+		"unsigned": true,
+		"unit": "seconds"
+	},
+	"relay_type_behavior": {
+		"label": "Relay Type",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Normally open (reports off when relay is open / on when relay is closed)",
```

Comment:
Please change to "Normally open (off when relay is open)" The other way can be inferred.